applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py

```diff
@@ -36,7 +36,7 @@ def Initialize(self):
 			self.RemeshingProcessMMG.ExecuteInitialize()
 
         # for the dem contact forces coupling
-        self.InitializeDummyNodalForces()
+		self.InitializeDummyNodalForces()
 
 		print(" /$$$$$$$$ /$$$$$$$$ /$$      /$$  /$$$$$$  /$$$$$$$  /$$$$$$$$ /$$      /$$")

```

Comment:
Use the logger

Reply:
now I use the logger but i don't know the difference :)